package.json

```diff
@@ -98,6 +98,7 @@
   },
   "dependencies": {
     "@aws-crypto/sha256-js": "^2.0.1",
+    "@aws-sdk/client-chime-sdk-messaging": "^3.0.0",
```

Comment:
So this is adding 1.5mb size to JS SDK. Just a thought: What if we take in a callback in `MessagingSessionConfiguration` and that provides us with updated URL? The `startConnecting` function can internally call this callback for getting the refreshed credentials and the endpoint.

Not a great builder experience as they have to now implement a callback in their app, but better than adding 1.5Mb size to JS SDK package? Not sure how many people are using messaging + meeting session and for those using only meeting session its an un-needed dependency.

cc: @ltrung @simmkyu

Reply:
I prefer the increase in size than adding a cumbersome callback. Also seem like you force builders to do a migration then.

Reply:
+1 with what ltrung.  You can't use the messaging sdk with out the aws sdk anyways.  In the future, modularization of the amazon-chime-sdk-js will make the concerns about package size moot